<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>htlChatClient</class>
 <widget class="QMainWindow" name="htlChatClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>htlChatClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editHost">
        <property name="text">
         <string>aboehler.at</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinPort">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>62304</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editUsername"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chkTls">
        <property name="text">
         <string>TLS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btnSend">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send File...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="editMessage"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblChatroom">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Chatroom: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTextBrowser" name="textData"/>
    </item>
    <item row="2" column="2">
     <widget class="QListWidget" name="listUserlist"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnPublicChatroom">
      <property name="text">
       <string>Public Chatroom</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
